.travis.yml

```diff
@@ -184,7 +184,7 @@ matrix:
         - CMD="mvn -e package -Passembly"
 
 script:
-  - SKIP_FILES=".github|appveyor.yml|circle.yml|distelli-manifest.yml|fast-forward-merge.sh|LICENSE|LICENSE.apache20|README.md|release.sh|RIGHTS.antlr|shippable.yml|shippable.sh|wercker.yml|wercker.sh|intellij-idea-inspections.xml"
+  - SKIP_FILES=".github|appveyor.yml|circle.yml|distelli-manifest.yml|fast-forward-merge.sh|LICENSE|LICENSE.apache20|README.md|release.sh|RIGHTS.antlr|shippable.yml|shippable.sh|wercker.yml|wercker.sh|intelij-idea-inspections.xml"
   - SKIP_CI=$(if [[ $(git diff --name-only HEAD HEAD~1 | grep -vE "$SKIP_FILES" | cat | wc -c | sed 's/^ *//' ) > 0 ]]; then echo false; else echo true; fi;)
```

Comment:
IntelliJ is correct:
https://www.jetbrains.com/idea/

Reply:
agree, should be reverted.

Reply:
Oops, will undo that side/fix to consistently use `IntelliJ IDEA` (and not other capitalization variations, accepting `intellij` for filenames). I did try to check that (we actually use their software), but apparently my eyes glazed over.